src/pages/Profile.js

```diff
@@ -1,52 +1,70 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Header } from "../components/Header";
```

Comment:
you can put all the import in one file so your code looks more cleaner

Reply:
Yes Thank you, but actually I dont have more than 2-3 files .